vcf/src/parse.rs

```diff
@@ -0,0 +1,31 @@
+use std::collections::HashMap;
+
+use crate::{Header, HeaderValue};
+
+impl<'src> Header<'src> {
```

Comment:
Does it not make sense to put these in the `header.rs` file? It seems like they are essentially constructors for the header objects.